acceptance/tests/environment/environment_scenario-non_existent.rb

```diff
@@ -46,9 +46,9 @@
   },
   :puppet_agent => {
     :exit_code => 1,
-    :matches => [%r{Warning.*404.*Could not find environment '#{env}'},
+    :matches => [%r{(Warning|Error).*(404|400).*Could not find environment '#{env}'},
```

Comment:
I'm a little surprised that we sometimes get a 400 here instead of a 404?  Do you know why, Eric?